drupal/fabfile.py

```diff
@@ -449,7 +449,7 @@ def main(repo, repourl, build, branch, buildtype, keepbuilds=10, url=None, fresh
       phpunit_tests_failed = common.Tests.run_phpunit_tests(repo, branch, build, phpunit_group, phpunit_test_directory, phpunit_path)
       if phpunit_fail_build and phpunit_tests_failed:
         Revert._revert_db(repo, branch, build)
```

Comment:
Should this be `Revert._revert_db(alias, branch, build)` instead? So replace `repo` with `alias`?